src/js/edu-benefits/components/military-history/MilitaryServiceTour.jsx

```diff
@@ -3,12 +3,25 @@ import React from 'react';
 import ErrorableCheckbox from '../../../common/components/form-elements/ErrorableCheckbox';
 import ErrorableTextInput from '../../../common/components/form-elements/ErrorableTextInput';
 import ErrorableTextarea from '../../../common/components/form-elements/ErrorableTextarea';
+import ErrorableDate from '../../../common/components/form-elements/ErrorableDate';
+import ErrorableCurrentOrPastDate from '../../../common/components/form-elements/ErrorableCurrentOrPastDate';
 import ExpandingGroup from '../../../common/components/form-elements/ExpandingGroup';
-import DateInput from '../../../common/components/form-elements/DateInput';
 
-import { validateIfDirtyDateObj, validateIfDirty, isNotBlank, isValidDateField, isValidDateRange } from '../../utils/validations';
+import { validateIfDirty, isNotBlank, isValidDateRange } from '../../utils/validations';
+import { isDirtyDate } from '../../../common/utils/validations';
 import ServicePeriodsReview from './ServicePeriodsReview';
 
+function validEndDate(tour) {
```

Comment:
I think I can change this around so it it's less complicated. This instance is more complicated mainly because it's not using a validateIfDirty-like helper function, which I can add.